modules/nw-ovn-kubernetes-rollback-live.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+// * networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-rollback-live_{context}"]
+= Using the live migration method to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can rollback to the OpenShift SDN Container Network Interface (CNI) network plugin by using the live migration method. During the migration with this method, nodes are automatically rebooted and service to the cluster is not interrupted.
+
+[IMPORTANT]
+====
+You must wait until the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
+====
+
```

Comment:
Maybe another Important info for the CU might be to inform the user that they can only migrate in 4.16 and no later version of OCP because OpenShift SDN will be depreciated. This will future proof your docs from 4.17 changes to reflect this reality.

Reply:
@martinkennelly @pliurh Does this mean that we'll only want *this specific procedure* in version 4.16? and we do *not* want it to appear in version 4.17+? That will change the target branch of this pull request.

Reply:
Yes. Live migration will only be available in version 4.16. OpenShiftSDN will be removed in version 4.17. Therefore, users cannot upgrade an SDN cluster to 4.17 without migrating to OVN beforehand. Both Live and offline migration chapters shall be removed from 4.17+ documents.